pkg/resource/deploy/import_test.go

```diff
@@ -101,7 +101,7 @@ func TestImporter(t *testing.T) {
 						Name: tokens.MustParseStackName("stack-name"),
 					},
 					source: &nullSource{},
-					providers: providers.NewRegistry(&mockHost{
+					providers: providers.NewRegistry(&plugin.MockHost{
```

Comment:
We had two mock host implementations. I've deleted the one in source_query_test and we can just use the public one from the plugin package now.